ERCS/erc-x.md

```diff
@@ -0,0 +1,324 @@
+
+The **PermaLink-ABTs** implementation takes the ABT model further by enforcing a permanent binding between one token and another—whether it's another ABT, NFT or NFKBT. This permanent binding ensures that tokens can be transferred as a single unit, reducing complexity and gas fees. Instead of relying on traditional minting, PermaLink-ABTs use a `reveal` mechanism, activating tokens from a predefined supply. This reduces gas costs and encourages efficient linking of assets, enabling greater composability across multiple sectors.
+
+PermaLink-ABTs consolidate asset value by allowing multiple subordinate tokens to be linked to a single binding token, providing enhanced composability and reducing fragmentation. By requiring only the binding token to be transferred, all associated assets move in sync, making it easier to manage portfolios and move groups of assets together. This approach fosters collaboration, value accrual, and compatibility across ecosystems, whether for digital identities, RWAs, NFTs, or other on-chain assets.
+
+
+## Specification
+
+### `IABT` (Token Interface)
+
+**NOTES**:
+
+- The following specifications use syntax from Solidity `0.8.27` (or above)
+- Callers MUST handle `false` from `returns (bool success)`. Callers MUST NOT assume that `false` is never returned!
+
+```solidity
+interface IABT {
+    event AssetBoundContractSet(address assetBoundContract);
+    event TokenRevealed(uint256 tokenId);
+
+    function ownerOf(uint256 tokenId) external view returns (address);
+    function tokenExists(uint256 tokenId) external view returns (bool);
+    function totalSupply() external view returns (uint256);
+    function balanceOf(address owner) external view returns (uint256);
+
+    function reveal(uint256[] calldata tokenIds) external payable;
+}
+```
+
+### **Interface events**
+
+#### `AssetBoundContractSet` event
+
+Emitted when the contract is deployed and bound to `assetBoundContract`
+
+```solidity
+event AssetBoundContractSet(address assetBoundContract);
+```
+
+#### `TokenRevealed` event
+
+Emitted when the `tokenId` is revealed
+
+```solidity
+event TokenRevealed(uint256 tokenId);
+```
+
+### **Interface functions**
+
+The functions detailed below MUST be implemented.
+
+#### `ownerOf` function
+
+Returns the owner of the NFT specified by the `tokenId`. Will read from the `assetBoundContract` the owner and return it.
+
+```solidity
+function ownerOf(uint256 tokenId) external view returns (address);
+```
+
+#### `tokenExists` function
+
+Returns true if the owner of the token is not the zero address (`0x0`)
```

Comment:
This is a very strict requirement. So the zero address is never allowed to own tokens? Would it make more sense to define this more vaguely in case something comes along in the future where the zero address is useful for some reason?